src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -1817,9 +1817,9 @@ protected internal virtual TypeDefinition MarkType (TypeReference reference, Dep
 			MarkCustomAttributes (type, new DependencyInfo (DependencyKind.CustomAttribute, type));
 			MarkSecurityDeclarations (type, new DependencyInfo (DependencyKind.CustomAttribute, type));
 
-			if (type.BaseType is not null &&
+			if (_context.TryResolve (type.BaseType) is TypeDefinition baseType &&
```

Comment:
Is nullable actually enabled in this file?

Reply:
Nullable is not enabled on MarkStep, but methods in Annotations.cs that have non-nullable parameters are flagged when called with a possible null. The warning is on callsite so the warning appears in MarkStep even if nullable has not been enabled.